packages/dara-core/dara/core/main.py

```diff
@@ -69,6 +71,10 @@
 )
 from dara.core.logging import LoggingMiddleware, dev_logger, eng_logger, http_logger
 
+# Inject the encoder to pydantic ENCODERS_BY_TYPE, which will be called in fastapi jsonable_encoder
+for key, value in encoder_registry.items():
+    ENCODERS_BY_TYPE[key] = value['serialize']
+
```

Comment:
Shouldn't this be called within the startup function so it is filled in after we fill our `encoder_registry` with user defined encoders?